evennia/typeclasses/models.py

```diff
@@ -126,6 +126,31 @@ class Meta:
         return new_class
 
 
+class DbHolder(object):
```

Comment:
@Griatch Done. NDBHolder was almost verbatim the same as DBHolder, so I consolidated them into DBHolder with an argument.